### DISCUSSION TITLE
Ideas for new packages

### DISCUSSION THREAD

**Discussion by sloorush** [aplteam/Tatin#216] [Ideas]
This is a space where people can note down and pick up ideas of packages that are needed by the community and what packages they can build.

Feel free to start a new thread with an idea of a package you want to build or a package you would like to have. Discussions can be continued in the thread of that post.

Please use one post per idea and look for duplicates before posting.

Thanks! <3

**Comment by sloorush** [comment]
## MongoDB

A package for MongoDB. MongoDB is a widely used database system and it would be nice to have an interface to MongoDB.

URL: https://www.mongodb.com/

**Comment by DavinChurch** [comment]
I am working on a package that gives us easy APL array-style access to ADO.Net, which can talk to most any database package. The basics are working fine but I'm having trouble trying to decide how best to deal with overlapping DB updates from multiple users. If anyone would like to contribute to the planning, please feel free to comment on this (or a new) thread.
